addon-images.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 D2iQ, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+aws-cpi-v1.27:
```

Comment:
How are we plan to consume this file?

Reply:
Great question! The idea is that any consumer (e.g. konvoy2) can run a simple `yq` query to get the list of images.

```
 yq '.*.images' addon-images.yaml
 ```
 
 or 
 
 ```
 yq '.*.images' addon-images.yaml | tr -d ' -'
 ```

Reply:
nice. each client wont have to build tooling to scrape images for bundling.